### PR TITLE
New account

### PR DESCRIPTION
**Pull request by ngocloan202**
New account UI and handle event

### CODE REVIEW

File: PharmacyManagement/HumanManage/NewAccount.cs [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        {
+            txtAccountID.Enabled = false;
+            ClearAllField();
+        }
+
+        private void btnAdd_Click(object sender, EventArgs e)
+        {
+            if (ValidateInput())
+            {
+                string sql = @"INSERT INTO ACCOUNT (Username, UserPassword, UserRole)
```
Comment (its-quan-ng) [inline PharmacyManagement/HumanManage/NewAccount.cs:29]: why dont we remove ID textbox? Either way, we dont use it

File: PharmacyManagement/Program.cs [hunk 2]
```diff
@@ -13,7 +14,7 @@ static void Main()
         {
             Application.EnableVisualStyles();
             Application.SetCompatibleTextRenderingDefault(false);
-            Application.Run(new Main());
+            Application.Run(new NewAccount());
```
Comment (its-quan-ng) [inline PharmacyManagement/Program.cs]: revert it

**Review by its-quan-ng — CHANGES_REQUESTED** [review]
I think we should add the case employeeID = null to display it on the datagrid

File: PharmacyManagement/HumanManage/NewAccount.cs (hunk 1 above)
Comment (ngocloan202) [reply PharmacyManagement/HumanManage/NewAccount.cs:29]: done

File: PharmacyManagement/Program.cs (hunk 2 above)
Comment (ngocloan202) [reply PharmacyManagement/Program.cs]: done

**Comment by ngocloan202** [comment]
I have fixed the form closing error

**Review by its-quan-ng — APPROVED** [review]

**Pull request merged**